python/xnvme-cy-header/aux/generate_cython_header.py

```diff
@@ -55,7 +55,6 @@ def generate_pxd(workspace_root):
         "libxnvme_znd.h",
         "libxnvme_pp.h",
         "libxnvme_be.h",
-        "libxnvme_sgl.h",
```

Comment:
There are traces of xnvme_sgl here (Line 9, 10). they can be removed

Reply:
Overlooked this, fixed now. Also, found another mention of the SGLs in the cython header. Removed that as well.

Reply:
Nice, to get rid of some sgl-specific fixes as well.